common/on-chip-calib.cpp

```diff
@@ -33,8 +33,8 @@ namespace rs2
             if( val_in_range( device_name_string, { std::string( "Intel RealSense D415" ) } ) )
                 speed = 4;
 
-            if ( strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCD") || strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCE"))
-                is_mipi_sku = true;
+            if ( strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCD") == 0 || strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCE") == 0)
+                host_assistance = true; // To be used for MIPI SKU only
```

Comment:
`host_assistance` is int, not bool. Can get values 0-3.

Reply:
Fixed